### PR TITLE
fix #302, wrong argument in usage in dummy-iiostream

### LINKED ISSUE #302: Wrong argument help in example.
In the examples/dummy-iiostream.c example, the help for the read method is 2 read, 3 read raw, while the order of the enum makes it 2 read raw, 3 read.

### PR DESCRIPTION
**Pull request by rgetz**
Signed-off-by: Robin Getz <robin.getz@analog.com>

### CODE REVIEW

File: examples/dummy-iiostream.c [hunk 1]
```diff
@@ -99,6 +99,13 @@ enum {
 	MAX_READ_METHOD,
 };
 static int buffer_read_method = BUFFER_POINTER;
+// these strings must be in the same order as the above enum
+static char read_method_str[MAX_READ_METHOD][20] = {
```
Comment (mhennerich) [inline examples/dummy-iiostream.c]: use:
static const char * const read_method_str[] = {

Comment (rgetz) [reply examples/dummy-iiostream.c]: changed

**Comment by dNechita** [comment]
There's a small conflict that needs to be handled.

**Comment by rgetz** [comment]
looks like I based this off an old unfixed/stale copy of master I had - master was fixed in August, and this patch isn't necessary (was fixed a different way). Sorry for the noise.

**Pull request closed without merging**